functorch/experimental/cond.py

```diff
@@ -115,6 +123,17 @@ def inner(pred, true_fn, false_fn, operands):
     return res
 
 
+@cond.py_impl(FakeTensorMode)
+def cond_fake_tensor_mode(pred, true_fn, false_fn, operands):
+    return true_fn(*operands)
```

Comment:
This is a bit fishy, I'd expect it to run true_fn(*operands), false_fn(*operands), and then assert that everything is identical